mkt/reviewers/tests/test_views.py

```diff
@@ -3986,7 +3986,7 @@ def setUp(self):
         self.grant_permission(self.profile, 'Apps:ModerateReview')
         self.create_switch('reviews-translate')
         user = user_factory(email='diego')
-        app = app_factory(app_slug='myapp')
+        app = app_factory(app_slug='myapp~@[]{},.#;:')
```

Comment:
I was going for what the regex allowed but actually looks like https://github.com/mozilla/zamboni/blob/master/mkt/site/utils.py#L172 limits it to just ~-_